Packages/com.bci4kids.bciessentials/Runtime/Scripts/Behaviors/BCIControllerBehavior.cs

```diff
@@ -495,6 +503,18 @@ public virtual void ReceiveMarkers()
                         // If there are square brackets then remove them
                         responseString = responseString.Replace("[", "").Replace("]","").Replace(".", "");
 
+                        //try to parse the rest of the response as an integer. Handle if it is the formate np.int64().
```

Comment:
I guess this is fo fix that new issue where there can be multiple predictions. From what I understand, this fixes the issue if there is is only one prediction, what happens if there are more? As a temp fix, seems fine